t/t0005-rexec.t

```diff
@@ -23,7 +23,7 @@ test_expect_success 'basic rexec functionality (process fail)' '
 
 test_expect_success 'basic rexec - cwd correct' '
 	(cd /tmp &&
-         cwd=`${FLUX_BUILD_DIR}/t/rexec/rexec pwd` &&
+         cwd=`${FLUX_BUILD_DIR}/t/rexec/rexec sh -c pwd` &&
```

Comment:
did this change get squashed into the wrong commit, it's in the "docker: make /tmp a symlink to /var/tmp " commit

Reply:
Hm, yeah that was `git commit -a` mistake. Thanks! I'll fixup your two comments and repush. Good thing Travis failed this time.